work-with-datetimes-strptime-and-strftime/dt_convert.py

```diff
@@ -4,12 +4,12 @@
 
 
 def years_ago(date):
-    """Receives a date string of 'DD MMM, YYYY', for example: 8 Aug, 2015
+   """Receives a date string of 'DD MMM, YYYY', for example: 8 Aug, 2015
        Convert this date str to a datetime object (use strptime).
        Then extract the year from the obtained datetime object and subtract
        it from the THIS_YEAR constant above, returning the int difference.
        So in this example you would get: 2018 - 2015 = 3"""
-    pass
+   return  THIS_YEAR - datetime.strptime(date, '%d %b, %Y').year
```

Comment:
could use an intermediate variable for `datetime.strptime(date, '%d %b, %Y').year`